### ISSUE TITLE
/ro default to user radius

### ISSUE THREAD

**Issue by Windows-i7** [TotalFreedom/TotalFreedomMod#399]
I have seen admins accidentally use ro in a way that removes from a radius of all players when they meant only a radius for themself. The command would be less accident prone if an extra parameter needed to be specified for all players (maybe a * for player parameter?)

**Comment by Wild1145** [comment]
I agree that this is a problem, normally it comes down to admins just not thinking, or admins not even being trained on how to even use the bloody commands...
